packages/camera/camera_platform_interface/lib/src/method_channel/method_channel_camera.dart

```diff
@@ -175,6 +175,16 @@ class MethodChannelCamera extends CameraPlatform {
         <String, dynamic>{'cameraId': cameraId},
       );
 
+  @override
+  Future<void> setFlashMode(int cameraId, FlashMode mode) =>
+      _channel.invokeMethod<void>(
+        'setFlashMode',
+        <String, dynamic>{
+          'cameraId': cameraId,
+          'mode': mode.toString().split('.').last,
```

Comment:
I'd recommend doing a `Map<FlashMode, String>` to do this conversion, instead of relying on the "toString" representation of the Enum.